content/reference/gql-types.md

```diff
@@ -0,0 +1,122 @@
+
+### Basic types
+
+Every manufacturing object in the Rhize database is made of fields that are of one the following basic types.
+In official GraphQL terminology, these types are called [_scalar types_](https://graphql.org/learn/schema/#scalar-types).
+
+- `String`
+- `Int`
+- `Float`
+- `Enum`
+- `Boolean`
+- `id`: A string representing a unique object within a defined [object type](#object-type).
+- `iid`: The object's unique address in the database. For example, `0xf9b49`.
+- `DateTime`: A timestamp in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format.
+- `Geo`: Geometry types for geo-spatial coordinates
+
+### Object type
+
+The preceding basic types form the building blocks for Rhize's manufacturing object schema, with data models corresponding to ISA-95.
+
+Each object is made of manufacturing-specific fields of one of the basic types.
+For example, the `materialActual`  object has basic fields, including:
+- `description`, a `String`.
+- `effectiveEnd`, a `DateTime`
+
+The `materialActual` also has complex fields describing associated manufacturing objects.
+For example, its fields include
+the array of associated `MaterialLot` objects, the `MaterialDefinition` object, and so on.
+All objects in the database have relationships to other objects.
+
+## Scalar filters
+
+Every field that has a basic type can be used as a filter.
+The filters that are available depend on the data type, but the behavior of the `String` filters corresponds closely to `DateTime` and `Int` filters.
+
+### String filters
+
+String properties have the following filters:
+
+| Filter                         | Description                                               | Example argument                                 |
+|--------------------------------|-----------------------------------------------------------|--------------------------------------------------|
+| `eq`                           | Equals (exact match)                                      | `(filter: {id: {eq: "match"}})`                  |
+| `in`                           | From this match of arrays                                 | `(filter: {id: {in: ["dough", "cookie_unit"]}})` |
+| `lt`,`gt`,`le`, `ge` `between` | Less than, greater than, or between a lexicographic range | `(filter: {id: {lt: "M"}`                        |
+| `regexp`                       | A regular expression match using [`RE2` syntax](https://github.com/google/re2/wiki/Syntax/)                | `(filter: {id: {regexp: "/hello/i"}})`           |
+| `anyoftext`                    | A match for any entered strings, separated by spaces      | `(filter: {id: {anyoftext: "100 MD"}})`          |
+| `alloftext`                    | A match for all entered strings                           | `(filter: {id {alloftext: "MD"}})`               |
+
+### Integers, floats, DateTimes
+
+Properties that have a type of `Int`, `Float`, or `DateTime` can be filtered by the following keywords.
```

Comment:
The same as above applies here, try filtering the `rules` of a `workflowSpecificaiton` based on its `delayTime` for example.